### ISSUE TITLE
Adium crashes on launch

### ISSUE THREAD

**Issue by andrews05** [majn/telegram-purple#239]
My friend is trying to use the Adium plugin (v1.2.5) with the current version of Adium (v1.5.10.1) on OS 10.8. It says the plugin installs successfully, but on relaunch Adium just quits unexpectedly. Tried clearing prefs and data but still won't work. I've attached a crash log. Any idea what's going on?

[telegram-adium crash.txt](https://github.com/majn/telegram-purple/files/131666/telegram-adium.crash.txt)

**Comment by majn** [comment]
Its probably due to OSX 10.8. (Note: I don't on old OSX versions).

If he needs to uninstall it, he can simply remove the plugin file from ~/Library/Application Support/Adium 2.0/PlugIns

**Comment by andrews05** [comment]
Yeah, I suspected the same. Do you plan to test/support anything prior to 10.11?

**Comment by majn** [comment]
If I find the cause of this bug and its possible I will fix it.

**Comment by andrews05** [comment]
Okay, thanks.

**Comment by r3t** [comment]
Can you try to set deployment target to 10.8? I think, it should help.
